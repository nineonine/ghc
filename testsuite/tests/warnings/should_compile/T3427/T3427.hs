module T3427 where

import A

foo :: Foo
foo = Foo

bar :: Bar
bar = Bar

baz :: Baz
baz = Baz
