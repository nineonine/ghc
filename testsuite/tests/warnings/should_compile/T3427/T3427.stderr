
T3427.hs:5:8: warning: [-Wdeprecations (in -Wdefault)]
    In the use of type constructor or class ‘Foo’ (imported from A):
    Deprecated: "Don't use type and data constructor Foo"

T3427.hs:6:7: warning: [-Wdeprecations (in -Wdefault)]
    In the use of data constructor ‘Foo’ (imported from A):
    Deprecated: "Don't use type and data constructor Foo"

T3427.hs:8:8: warning: [-Wdeprecations (in -Wdefault)]
    In the use of type constructor or class ‘Bar’ (imported from A):
    Deprecated: "Don't use type Bar"

T3427.hs:12:7: warning: [-Wdeprecations (in -Wdefault)]
    In the use of data constructor ‘Baz’ (imported from A):
    Deprecated: "Don't use data constructor Baz"
